client/src/components/rounds/Apply.tsx

```diff
@@ -41,18 +55,32 @@ function Apply() {
     }
   }, [dispatch, props.id, props.round]);
 
-  if (props.status === Status.Error) {
-    return <div>Error: {props.error}</div>;
+  if (props.roundStatus === RoundStatus.Error) {
+    return <div>Error loading round data: {props.roundError}</div>;
   }
 
-  if (props.status !== Status.Loaded) {
+  if (props.roundStatus !== RoundStatus.Loaded) {
     return <div>loading...</div>;
   }
 
   if (props.roundState === undefined || props.round === undefined) {
     return <div>something went wrong</div>;
   }
 
+  if (props.applicationStatus === ApplicationStatus.Error) {
```

Comment:
Maybe we can move these into toast components like the project creation form?

Reply:
good idea I added this issue https://github.com/gitcoinco/grant-hub/issues/198